docker-compose.yml

```diff
@@ -38,7 +38,7 @@ services:
         max-file: "10"
 
   cardano-db-sync:
-    image: inputoutput/cardano-db-sync:10.0.0
+    image: inputoutput/cardano-db-sync:10.0.1
```

Comment:
This file is updated on release. If you look at the `release/10.0.x` branch you will see that this has been updated.

`db-sync` should only really be run from release branches where this value is always correct.